src/mod_filtering.c

```diff
@@ -218,10 +218,16 @@ static int setup_filtergraph(struct filtering_ctx *ctx)
         snprintf(args, sizeof(args), "time_base=%d/%d:sample_rate=%d:sample_fmt=%s",
                  time_base.num, time_base.den, codecpar->sample_rate,
                  av_get_sample_fmt_name(codecpar->format));
+#if LIBAVUTIL_VERSION_INT < AV_VERSION_INT(57, 24, 100)
```

Comment:
Ah also please mention the support for recent FFmpeg in the changelog

Reply:
Done.